orbit/pkg/table/extension.go

```diff
@@ -41,6 +41,12 @@ type Extension interface {
 // Opt allows configuring a Runner.
 type Opt func(*Runner)
 
+// Global variables for osquery extension manager client and logger for Kolide tables
+var (
+	serverClient *osquery.ExtensionManagerClient //nolint:unused
```

Comment:
We need to understand why some Kolide tables need a `osquery.ExtensionManagerClient`.